km_api/know_me/models.py

```diff
@@ -664,14 +664,10 @@ class ProfileTopic(mixins.IsAuthenticatedMixin, models.Model):
         name (str):
             The topic's name.
     """
-    GROUPED = 1
-    PAGED = 2
-    TEXT = 3
-    VISUAL = 4
+    TEXT = 1
```

Comment:
We shouldn't change the number associated with each type of row. It doesn't particularly matter now since we don't have any important data stored, but this would break any existing data.

If we change it back, we should add a comment explaining why we use these "arbitrary" numbers.